docs/_extra/api-reference/index.html

```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Hypothesis API documentation</title>
+    <title>Hypothesis API documentation (v1)</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
   </head>
   <body>
-    <redoc spec-url="hypothesis.yaml"></redoc>
-    <script src="https://cdn.jsdelivr.net/npm/redoc@2.0.0-alpha.17/bundles/redoc.standalone.js"></script>
+    <redoc spec-url="hypothesis-v1.yaml" expand-responses="200,201"></redoc>
```

Comment:
`hypothesis.yaml` is still in the source tree. Should it be removed? I found one remaining reference to it in `docs/developing/documentation.rst`.